### ISSUE TITLE
Allow admins of headless sessions to edit permissions and use the join verifier

### ISSUE THREAD

**Issue by scrapSavage** [Yellow-Dog-Man/Resonite-Issues#1010]
### Is your feature request related to a problem? Please describe.

Currently headless sessions that host worlds that already have permissions set (like allowing cloud avatars) can have these custom permissions setup, they also cannot enable or take advantage of the custom join verifier.

### Describe the solution you'd like

Allow admins in headless sessions to edit world permissions and enable join verifiers.

### Describe alternatives you've considered

The only way to do this is through mods or creating a world before putting it live on a headless.

**Comment by Frooxius** [comment]
This won't happen until hard permissions are implemented. Right now we don't have mechanisms to do this securely, so it's limited specifically to the host.

**Comment by art0007i** [comment]
@Frooxius 
actually while the permission components will probably have to stay host only until hard permissions, the join verifier could easily be made accessible for admins, as internally it requires the same permission that allows you to edit for example the world's name. 

I've made a mod that allows you to enable the custom join verifier as an admin and it works perfectly. https://github.com/art0007i/CustomJoinVerifierFix

**Comment by art0007i** [comment]
yeah allowing admins to change the join verifier would still maintain security.
